modules/assim.sequential/R/sda.enkf.R

```diff
@@ -515,11 +512,15 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL) {
       ### no process variance -- forecast is the same as the analysis ###
       if (processvar==FALSE) {
         mu.a <- mu.f
-        Pa   <- Pf + Q #Where are we getting Q here? Shouldn't it be just Pf?
+        Pa   <- Pf + Q
         ### yes process variance -- no data
       } else {
         mu.a <- mu.f
-        Pa   <- Pf + solve(q.bar)#Where are we getting q.bar here?
+        if(is.null(q.bar)){
+          q.bar <- diag(ncol(X))
```

Comment:
Won't this set the process variance to 1? That doesn't seem uninformative. Maybe we should set to a matrix of all 0's so you get Pa = Pf + 0?